### PR TITLE
new domain

### PR DESCRIPTION
**Pull request by 2M4U**

### CODE REVIEW

File: domains/fbi.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "2Much4U's Website",
+  "repo": "https://github.com/2m4u",
+  "owner": {
+    "username": "2M4U",
+    "email": "2m4u@fbi.is-a.dev"
+  },
+  "record": {
+    "CNAME": "2m4u.netlify.app"
```
Comment (TheBotlyNoob) [inline domains/fbi.json:9]: Netlify/Vercel will not work with a `CNAME` record, consider switching to [Railway](https://railway.app/?referralCode=jj), or use a `URL` record.

**Review by TheBotlyNoob — DISMISSED** [review]

File: domains/fbi.json (hunk 1 above)
Comment (2M4U) [reply domains/fbi.json:9]: I tried to deploy on [Railway](https://railway.app/?referralCode=jj) however it crashes so... I think imma change it to `URL` record if that helps?

File: domains/fbi.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "2Much4U's Website",
+  "repo": "https://github.com/2m4u",
+  "owner": {
+    "username": "2M4U",
+    "email": "2m4u@fbi.is-a.dev"
```
Comment (MaskDuck) [inline domains/fbi.json:6]: maybe not use an is-a-dev ẹmail here, the email is used to call you when there is a problem with the domain, if your domain is broken how should we contact you?

**Review by MaskDuck — CHANGES_REQUESTED** [review]

**Review by MaskDuck — APPROVED** [review]

**Comment by TheBotlyNoob** [comment]
Uhh the checks are being weird

**Comment by MaskDuck** [comment]
sorry due to ci being weird we cant merge this, i would merge it tomorrow when i wake up

**Pull request merged**